### ISSUE TITLE
"lib/graal_isolate.h: Operation not permitted" installing native-image using graalvm 20.1.0

### ISSUE THREAD

**Issue by kemitix** [rinx/setup-graalvm-ce#2]
```
Run rinx/setup-graalvm-ce@v0.0.1
/usr/bin/docker run --name d309c8142601d34f81bf62d6d40e4cee0b_141ee6 --label 3888d3 --workdir /github/workspace --rm -e INPUT_GRAALVM-VERSION -e INPUT_JAVA-VERSION -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true --network github_network_4756c82f43534e4c9c0fb58458ab3853 -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/naolo/naolo":"/github/workspace" 3888d3:09c8142601d34f81bf62d6d40e4cee0b  "20.1.0" "java11"
Install GRAALVM 20.1.0 java11
setup-native-image
1s
    JAVA_HOME: /home/runner/work/_temp/_github_home/graalvm
Run gu install native-image
  gu install native-image
  shell: /bin/bash -e {0}
  env:
    PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/runner/work/_temp/_github_home/graalvm/bin
    GRAALVM_HOME: /home/runner/work/_temp/_github_home/graalvm
    GRAALVM_VERSION: 20.1.0
    JAVA_HOME: /home/runner/work/_temp/_github_home/graalvm
Downloading: Component catalog from www.graalvm.org
Processing Component: Native Image
Downloading: Component native-image: Native Image  from github.com
[                    ]
[#                   ]
[##                  ]
[###                 ]
[####                ]
[#####               ]
[######              ]
[#######             ]
[########            ]
[#########           ]
[##########          ]
[###########         ]
[############        ]
[#############       ]
[##############      ]
[###############     ]
[################    ]
[#################   ]
[##################  ]
[################### ]
[####################]
Installing new component: Native Image (org.graalvm.native-image, version 20.1.0)
Installation of Native Image failed: /home/runner/work/_temp/_github_home/graalvm/lib/graal_isolate.h: Operation not permitted
I/O error occurred: /home/runner/work/_temp/_github_home/graalvm/lib/graal_isolate.h: Operation not permitted
##[error]Process completed with exit code 2.
```

**Comment by rinx** [comment]
@kemitix Thanks for your report.
I'm going to find out what's the problem and fix it soon.

**Comment by kemitix** [comment]
Thanks for the quick turn around.

I've tried using the new version, but get a different error
```
##[warning]Unexpected input 'native-image', valid inputs are ['entryPoint', 'args', 'graalvm-version', 'java-version']
Run rinx/setup-graalvm-ce@v0.0.2
/usr/bin/docker run --name d3bb33e26482454e1499764ea2d59323d6_894de5 --label 3888d3 --workdir /github/workspace --rm -e INPUT_GRAALVM-VERSION -e INPUT_JAVA-VERSION -e INPUT_NATIVE-IMAGE -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true --network github_network_1cfdca9dc72c46de9a4ab6d6c59b6ec7 -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/naolo/naolo":"/github/workspace" 3888d3:bb33e26482454e1499764ea2d59323d6  "20.1.0" "java11"
Install GRAALVM 20.1.0 java11
chown: unknown user/group runner:runner
```

Full logs: https://github.com/kemitix/naolo/pull/222/checks?check_run_id=750436260
Configuration: https://github.com/kemitix/naolo/blob/510cd9ab6c27135f2849d153cc90ee52933a67a8/.github/workflows/maven-build.yml

**Comment by rinx** [comment]
@kemitix Sorry for belated response.
I've just published v0.0.3 for fixing it. (sorry, v0.0.2 is accidentally released)

Now you can specify `native-image` parameter to install native-image command.
```yaml
      - name: setup-graalvm-ce
        uses: rinx/setup-graalvm-ce@v0.0.3
        with:
          graalvm-version: "20.1.0"
          java-version: "java11"
          native-image: "true"
```

Could you please try it?

**Comment by kemitix** [comment]
:+1:  `rinx/setup-graalvm-ce@v0.0.3` Works.
Thanks!
